### PR TITLE
v3rpc: add 'active' gRPC streamsGauge

### PR DESCRIPTION
**Pull request by gyuho**
@heyitsanthony @xiang90

### CODE REVIEW

File: etcdserver/api/v3rpc/metrics.go [hunk 1]
```diff
@@ -33,6 +33,14 @@ var (
 			Help:      "Counter of failed requests.",
 		}, []string{"grpc_service", "grpc_method", "grpc_code"})
 
+	activeGauage = prometheus.NewGaugeVec(
```
Comment (xiang90) [inline etcdserver/api/v3rpc/metrics.go]: probably just streamsGauage

File: etcdserver/api/v3rpc/metrics.go [hunk 2]
```diff
@@ -33,6 +33,14 @@ var (
...
+		prometheus.GaugeOpts{
+			Namespace: "etcd",
+			Subsystem: "grpc",
+			Name:      "active_streams_total",
```
Comment (xiang90) [inline etcdserver/api/v3rpc/metrics.go]: streams_total

File: etcdserver/api/v3rpc/metrics.go [hunk 3]
```diff
@@ -33,6 +33,14 @@ var (
...
+			Help:      "Counter of active streams.",
```
Comment (xiang90) [inline etcdserver/api/v3rpc/metrics.go]: Number of active streams.

Comment (xiang90) [inline etcdserver/api/v3rpc/metrics.go]: this is not a counter. counter only increases.

**Comment by xiang90** [comment]
We need to add this into the metrics doc too.

**Comment by gyuho** [comment]
@xiang90 Just updated doc as well. PTAL. Thanks.

Will redeploy test dashboard cluster once this gets merged.

**Comment by xiang90** [comment]
change streams to active_streams. then lgtm

**Pull request merged**